src/app/containers/ArticleMain/index.stories.jsx

```diff
@@ -5,10 +5,13 @@ import { withKnobs } from '@storybook/addon-knobs';
 import { ServiceContextProvider } from '../../contexts/ServiceContext';
 import { ToggleContextProvider } from '../../contexts/ToggleContext';
 import { RequestContextProvider } from '../../contexts/RequestContext';
-
 import ArticleMain from '.';
 import { articleDataNews, articleDataPersian } from '../Article/fixtureData';
 
+// article c5jje4ejkqvo contains a Headline, a Paragraph, a timestamp
+// a Portrait Image with Caption, a Landscape Image with Caption and Square Image with Caption.
+import articleData from '../../../../data/news/articles/c5jje4ejkqvo';
+
 // Not all services have fixtures for article data yet
 // the service selector will be constrained to services that have article fixtures:
 const availableFixtures = {
```

Comment:
I suggest changing 
`storiesOf('Containers|Article/Article Main', module)`
to 
`storiesOf('Containers|Main|Article', module)`

Same for the other Main Storybook examples:
`storiesOf('Containers|Main|Front Page', module)`
`storiesOf('Containers|Main|Media Page', module)`

This will group all of the main component stories in one place, similar to how the `Pages` have been grouped. 
<img width="199" alt="Screenshot 2019-09-11 at 11 24 48" src="https://user-images.githubusercontent.com/3028997/64689717-c8bc4a80-d486-11e9-834c-09fe25149f07.png">